tests/php/Forms/HTMLEditor/HTMLEditorSanitiserTest.php

```diff
@@ -43,6 +43,24 @@ public function testSanitisation()
                 '<p default1="specific1" force1="specific1">Test</p>',
                 '<p default1="specific1" force1="force1" default2="default2" force2="force2">Test</p>',
                 'Default attributes are set when not present in input, forced attributes are always set'
+            ),
+            array(
+                'a[href|target|rel]',
+                '<a href="/test" target="_blank">Test</a>',
+                '<a href="/test" target="_blank" rel="noopener, noreferrer">Test</a>',
```

Comment:
Could you please set the config value for `SilverStripe\Forms\HTMLEditor\HTMLEditorSanitiser.link_noopener_value` in this tests setup (that is, before or immediately after the assignment of the `$tests` array)?
It is not inconceivable for a project (or an extension) to alter this value via configuration, which would make this test fail. Best to enforce the known value that is tested for first :)

Reply:
Have updated

Reply:
Thanks! I have a comment about this, but I'll open a new thread since this is done :)